### ISSUE TITLE
Sometimes clan backyard bosses are way too overtuned

### ISSUE THREAD

**Issue by SharpTony** [project-rubika/issues#890]
### Reproduction steps

```bash
1.  Go to clan backyards
2.  Find the boss in that back corner
3.  Check his friendly nanos and .stats
```

### Expected results

```bash
The boss should be level 4 and his hp/buffs/etc should reflect that
```

### Actual results

```bash
The mob had like 10k hp, hit for 70-ish dmg, and had a damage shield that hit for 70-ish each hit.  This is not the norm, as I've gone to other backyards and the bosses were mostly fine.
```

### Bug prevalence

Occasionally

### Description

```bash
There have been instances where I've also seen the bosses only drop 1 item, but that needs further testing.
```

### Report link

i dunno what to put here

**Comment by kajoriEU** [comment]
#858

**Comment by kajoriEU** [comment]
fixed with #858
